hrm-domain/hrm-service/service-tests/contact-job/jobTypes/scrubTranscripts.test.ts

```diff
@@ -278,4 +291,91 @@ describe('Scrub job complete', () => {
     );
     expect(await pullDueContactJobs(new Date(), 5)).toHaveLength(0);
   });
+
+  test('Receive a completed retrieve transcript job and create a scrub job', async () => {
+    await contactApi.addConversationMediaToContact(
+      accountSid,
+      testContactId.toString(),
+      [
+        {
+          storeType: 'S3',
+          storeTypeSpecificData: {
+            type: S3ContactMediaType.SCRUBBED_TRANSCRIPT,
+            location: { bucket: 'mock-bucket', key: 'mock-new-scrubbed-transcript-path' },
+          },
+        },
+      ],
+      ALWAYS_CAN,
+      true,
+    );
+
+    await createContactJob()({
+      resource: await contactApi.getContactById(accountSid, testContactId, ALWAYS_CAN),
+      jobType: ContactJobType.SCRUB_CONTACT_TRANSCRIPT,
```

Comment:
The completed job you create should be a RETRIEVE_CONTACT_TRANSCRIPT job no? Receiving a completed retrieve transcript job is what starts the scrub transcript process